binr/radiff2/radiff2.c

```diff
@@ -76,8 +76,11 @@ static bool virgin = true;
 static void readstr(char *s, int sz, const ut8 *buf, int len) {
 	int die = 0;
 	int last = R_MIN (len, sz);
+	if (last < 0) {
```

Comment:
and you should *s = 0; in that case, otherwise the string is non-null terminated